lib/scala/logging-utils-akka/src/main/java/org/enso/logger/akka/AkkaConverter.java

```diff
@@ -0,0 +1,55 @@
+package org.enso.logger.akka;
+
+import static org.slf4j.event.Level.*;
+
+import java.util.Optional;
+import org.slf4j.event.Level;
+
+public class AkkaConverter {
+
+  /**
+   * Converts SLF4K's Level to an Akka one.
+   *
+   * @param level a SLF4J's Level to convert
+   * @return an equivalnet of `level` in Akka's LogLevel
+   */
+  public static akka.event.Logging.LogLevel toAkka(Level level) {
+    switch (level) {
+      case ERROR:
+        return new akka.event.Logging.LogLevel(1); // Error
```

Comment:
Could we try
```suggestion
        return new akka.event.Logging.ErrorLevel.asInt();
```
?
(I tried it in Scala, not sure if it will work OK in Java)

If possible, I'd prefer to use references from library rather than numeric constants + comments.

Reply:
Won't work like that because `LogLevel` is a value class